documentation/manual/src/main/asciidoc/en-US/modules/neo4j.asciidoc

```diff
@@ -784,6 +755,22 @@ tx.commit();
 ----
 ====
 
+In the case of JTA, Hibernate OGM attaches the Neo4J internal transaction to the JTA
+transaction lifecycle.
+That way when the JTA transaction is committed or rollbacked (for example by an EJB CMT or
+explicitly), the Neo4J transaction is also committed or rollbacked.
+This makes for a very nice integration in a Java EE container.
+
+[CAUTION]
+----
+This is NOT a true JTA/XA integration but more a lifecycle alignment:
+changes on more than one datasource won't be executed as a single atomic transaction.
+
+In particular, if the JTA transaction involves multiple resources, Neo4j might commit
```

Comment:
Couldn't we make sure somehow that Neo4j is committed last, something like the "[last resource gambit](https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/5/html/Administration_And_Configuration_Guide/lrco-overview.html)"?